### PR TITLE
Windows test

### PR DESCRIPTION
**Pull request by PokhodenkoSA**
Closes #10

### CODE REVIEW

**Comment by PokhodenkoSA** [comment]
Linux works too. Ready to merge.

**Comment by diptorupd** [comment]
@PokhodenkoSA Can you update the Readme with instructions on how to build on Windows? The old build_for_develop.bat does not seem to work.

**Comment by PokhodenkoSA** [comment]
> @PokhodenkoSA Can you update the Readme with instructions on how to build on Windows? The old build_for_develop.bat does not seem to work.

See #14

File: oneapi_wrapper/source/dppl_oneapi_interface.cpp [hunk 1]
```diff
@@ -106,7 +106,13 @@ class DpplOneAPIRuntimeHelper
     friend dppl::DpplOneAPIRuntime;
 };
 
-DpplOneAPIRuntimeHelper gRtHelper;
+DpplOneAPIRuntimeHelper& get_gRtHelper()
```
Comment (diptorupd) [inline oneapi_wrapper/source/dppl_oneapi_interface.cpp:109]:
```suggestion
// This singleton function is needed to create the DpplOneAPIRuntimeHelper object in a predictable manner
//  without which there is a chance of segfault.
DpplOneAPIRuntimeHelper& get_gRtHelper()
```

**Pull request merged**